src/styles/styles.css

```diff
@@ -0,0 +1,193 @@
+body {
```

Comment:
Do you still use this file?
As I see you split your style into smaller files and it seems that this file is leftover.

Reply:
Can I just remove these files and leave only SCSS files?